### PR TITLE
CT-987 prepare docs for external buckets

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira issue(s): N/A

Changes:

- make it easier to understand

---

### CODE REVIEW

**Review by romanbracinik — COMMENTED** [review]
Nehodime tam rovno este info o tych externych tabulkach ked je to uz nasadene?

**Comment by tomasfejfar** [comment]
@romanbracinik tohle nestačí? 
image

**Comment by romanbracinik** [comment]
> @romanbracinik tohle nestačí? image

tak ono som tam pridal este tu hlasku u BQ `Cloud Storage external tables
In case the registered dataset contains some [external tables](https://cloud.google.com/bigquery/docs/external-data-cloud-storage#required_roles), you need to add access to the target project to the source data in GCS. There are two ways:

Assign the storage.buckets.get permission to the project email sapi-9752@261928107051.iam.gserviceaccount.com for the entire source GCS bucket.
Assign the storage.objects.get permission to the project email sapi-9752@261928107051.iam.gserviceaccount.com for each object in the source GCS bucket, that serves as the source of the external table.`  a treba nagrantovat prava aj tomu filu z ktoreho je ta externa tabulka tak mi prislo fajn to tam spomenut.

**Comment by romanbracinik** [comment]
@tomasfejfar ale kludne to pak upravim, len ma napadlo ze napisem ked to este nieje mergnute.

**Comment by tomasfejfar** [comment]
Jo, tím pádem je to potřeba celé prescreenshotovat :( Ufff. Tak to bych nechal asi na další iteraci ještě.

BTW: Dobře se to screenshotovalo tak, že jsem tomu .dialog nastavil padding 40px a pak udělal node screenshot.

File: _includes/private-beta-warning.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="clearfix"></div><div class="alert alert-warning">
+    <p><strong>Private Beta Warning:</strong><br>
+    This feature is currently in private beta, some aspects of the feature may change or break unexpectedly. If you want to help us test it, please contant support. Please provide feedback using the feedback button in your project.</p>
```
Comment (jirkasemmler) [inline _includes/private-beta-warning.html:3]:
```suggestion
    This feature is currently in private beta, some aspects of the feature may change or break unexpectedly. If you want to help us test it, please contant support. Please provide feedback using the feedback button in your project.</p>
```
```suggestion
    This feature is currently in private beta, some aspects of the feature may change or break unexpectedly. If you want to help us test it, please contact [support](/management/support). Please provide feedback using the feedback button in your project.</p>
```

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
malickost

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**